packages/apps/docs/src/components/Markdown/styles.css.ts

```diff
@@ -29,3 +38,361 @@ globalStyle(
+    paddingBottom: 0,
+    marginLeft: vars.sizes.$1,
+    marginRight: vars.sizes.$1,
+  },
+);
+// P
+
+globalStyle(
+  `article
+  ${getClassName(paragraphWrapperClass)}
+  `,
+  {
+    margin: 0,
+  },
+);
+
+globalStyle(
+  `article
+  ${getClassName(paragraphWrapperClass)} +
+  ${getClassName(paragraphWrapperClass)}
+  `,
+  {
+    marginTop: vars.sizes.$md,
+  },
+);
+
+// Blockquote
+
+globalStyle(
+  `article
+  ${getClassName(blockquote)} p
+  `,
+  {
+    fontSize: 'inherit',
+    whiteSpace: 'pre-wrap',
+  },
+);
+
+globalStyle(
+  `article
+  ${getClassName(paragraphWrapperClass)} +
+  ${getClassName(paragraphWrapperClass)}
+  `,
+  {
+    marginTop: vars.sizes.$md,
+  },
+);
+
+// NOTIFICATION PARAGRAPHS
+
+globalStyle(
+  `article
+  ${getClassName(notificationWrapperClass)}
+  ${getClassName(paragraphWrapperClass)}
+  `,
+  {
+    margin: 0,
+  },
+);
+
+globalStyle(
+  `article
+  ${getClassName(notificationWrapperClass)}
+  ${getClassName(paragraphWrapperClass)}:first-of-type p
+  `,
+  {
+    marginTop: '0 !important',
+  },
+);
+
+globalStyle(
+  `article
+  ${getClassName(notificationWrapperClass)}
+  ${getClassName(paragraphWrapperClass)} :not(p:empty)
+  `,
+  {
+    marginTop: 20,
+  },
+);
+
+// UL | OL > LI
+
+globalStyle(
+  `article ul > li,
+  article ol > li`,
+  {
+    margin: '0 !important',
+    paddingTop: vars.sizes.$2xs,
+    paddingBottom: vars.sizes.$2xs,
+  },
+);
+
+globalStyle(
+  `article
+  ul > li
+  ${getClassName(codeWrapper)},
+  article
+  ol > li
+  ${getClassName(codeWrapper)}
+  `,
+  {
+    margin: 0,
+    marginTop: vars.sizes.$md,
+    marginBottom: vars.sizes.$lg,
+  },
+);
+
+globalStyle(
+  `article
+  ul +
+  ${getClassName(paragraphWrapperClass)},
+  article
+  ol +
+  ${getClassName(paragraphWrapperClass)}
+  `,
+  {
+    marginTop: vars.sizes.$md,
+  },
+);
+
+// A
+
+globalStyle(`${getClassName(treeListClass)} li:first-of-type`, {
+  marginTop: vars.sizes.$2,
+});
+
+// FIGURE
+
+globalStyle(
+  `article
+  ${getClassName(paragraphWrapperClass)} +
+  figure
+  `,
+  {
+    marginTop: vars.sizes.$4,
+  },
+);
+
+// TABLE
+
+globalStyle(`${getClassName(tableWrapperClass)} tr td`, {
+  verticalAlign: 'top',
+});
+
+globalStyle(`${getClassName(tableWrapperClass)} tr`, {
+  background: vars.colors.$white,
+});
+
+globalStyle(`${darkThemeClass} ${getClassName(tableWrapperClass)} tr`, {
+  background: vars.colors.$black,
+});
+
+globalStyle(`${getClassName(tableWrapperClass)} tr:nth-child(even)`, {
+  background: vars.colors.$gray20,
+});
+globalStyle(
+  `${darkThemeClass} ${getClassName(tableWrapperClass)} tr:nth-child(even)`,
+  { background: vars.colors.$gray90 },
+);
+
+globalStyle(`${getClassName(tableWrapperClass)} tr:hover`, {
+  background: vars.colors.$blue10,
+});
+globalStyle(`${darkThemeClass} ${getClassName(tableWrapperClass)} tr:hover`, {
+  background: vars.colors.$blue100,
+});
+
+// NAVIGATION
+
+globalStyle(
+  `${getClassName(treeListClass)} > li > ul > li > ul > li,
```

Comment:
this does look not excellent to me, but if it works it works

Reply:
It is awful. It was going to take too much time, so I left this for @sstraatemans. Also because, it was only for adding spaces around the third-level nav items, not breaking any functionality. This should be fixed in the tree list component.
Marked it as TODO, Thanks Jesse 👍

Reply:
fixed